modules/nw-egress-router-about.adoc

```diff
@@ -21,26 +36,44 @@ The egress router image is not compatible with Amazon AWS, Azure Cloud, or any o
 
-In _redirect mode_, an egress router pod sets up iptables rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
+In _redirect mode_, an egress router pod configures `iptables` rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
 
+ifdef::openshift-sdn[]
 In _HTTP proxy mode_, an egress router pod runs as an HTTP proxy on port `8080`. This mode only works for clients that are connecting to HTTP-based or HTTPS-based services, but usually requires fewer changes to the client pods to get them to work. Many programs can be told to use an HTTP proxy by setting an environment variable.
 
 In _DNS proxy mode_, an egress router pod runs as a DNS proxy for TCP-based services from its own IP address to one or more destination IP addresses. To make use of the reserved, source IP address, client pods must be modified to connect to the egress router pod rather than connecting directly to the destination IP address. This modification ensures that external destinations treat traffic as though it were coming from a known source.
 
 Redirect mode works for all services except for HTTP and HTTPS. For HTTP and HTTPS services, use HTTP proxy mode. For TCP-based services with IP addresses or domain names, use DNS proxy mode.
+endif::openshift-sdn[]
+
+ifdef::ovn[]
+[NOTE]
+====
+The egress router CNI plug-in supports _redirect mode_ only. This is a difference with the egress router implementation that you can deploy with OpenShift SDN. Unlike the egress router for OpenShift SDN, the egress router CNI plug-in does not support _HTTP proxy mode_ or _DNS proxy mode_.
+====
+endif::ovn[]
 
 [id="nw-egress-router-about-router-pod-implementation_{context}"]
 == Egress router pod implementation
 
-The egress router pod setup is performed by an initialization container. That container runs in a privileged context so that it can configure the macvlan interface and set up `iptables` rules.
-After the initialization container finishes setting up the `iptables` rules, it exits.
-Next the egress router pod executes the container to handle the egress router traffic. The image used varies depending on the egress router mode.
+ifdef::openshift-sdn[]
+The egress router pod setup is performed by an initialization container. That container runs in a privileged context so that it can configure the macvlan interface and set up `iptables` rules. After the initialization container finishes setting up the `iptables` rules, it exits. Next the egress router pod executes the container to handle the egress router traffic. The image used varies depending on the egress router mode.
```

Comment:
Possible minor tweaks for clarity (these are based on hunches, not rules, and I'm not certain I've retained the correct meaning throughout):

> The initialization container exits the egress router pod after it finishes setting up the `iptables` rules. The pod then executes the initialization container to handle the egress router traffic. The container image that is used varies depending on the egress router mode.

Reply:
Can I punt on this one?  What is written matches my understanding, but I wrapped the lines and didn't work on the SDN feature.

Reply:
Absolutely.